generate-fixture.js

```diff
@@ -0,0 +1,28 @@
+const { writeFileSync, readdirSync } = require('fs');
+const path = require('path');
+
+function createProjectFixtures() {
+  const projectDir = readdirSync(
+    path.join(__dirname, './src/projects'),
+    'utf-8'
+  );
+
+  let projectObject = { projects: {} };
+
+  projectDir.forEach((project) => {
+    // Remove excessive files
+    if (project === 'local-test' || project === 'index.pug') {
+      return;
+    }
```

Comment:
Are index.pug and local-test still used?  If not, we might as well remove them and this code.

Reply:
We will do the client up of the test directory in an other PR. So this could be changed then.